sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/DeepLinkActivity.java

```diff
@@ -25,14 +26,19 @@ public void onCreate(Bundle savedInstanceState) {
         String requestId = intent.getStringExtra(EXTRA_REQUEST_ID);
         String distributionGroupId = intent.getStringExtra(EXTRA_DISTRIBUTION_GROUP_ID);
         String updateToken = intent.getStringExtra(EXTRA_UPDATE_TOKEN);
+        String updateSetupFailed = intent.getStringExtra(EXTRA_UPDATE_SETUP_FAILED);
         MobileCenterLog.debug(LOG_TAG, getLocalClassName() + ".getIntent()=" + intent);
         MobileCenterLog.debug(LOG_TAG, "Intent requestId=" + requestId);
         MobileCenterLog.debug(LOG_TAG, "Intent distributionGroupId=" + distributionGroupId);
         MobileCenterLog.debug(LOG_TAG, "Intent updateToken passed=" + (updateToken != null));
+        MobileCenterLog.debug(LOG_TAG, "Intent updateSetupFailed passed=" + (updateSetupFailed != null));
 
         /* Store redirection parameters if both required values were passed. */
         if (requestId != null && distributionGroupId != null) {
             Distribute.getInstance().storeRedirectionParameters(requestId, distributionGroupId, updateToken);
+        } else if (updateSetupFailed != null){
+            // Otherwise just store error message to show update failure dialog in future.
```

Comment:
There is a code style rule that
1. you need to use `/* */` instead of `//` for explanation of the code.
2. add empty line before comment.
3. etc...

Reply:
Got it @jaelim-ms, thanks for feedback! I'll address this and other comments tomorrow and also will work on adding tests!